### ISSUE TITLE
'nimplex/src/nimplex/utils/stitching.nim' not being generated

### ISSUE THREAD

**Issue by amr8004** [amkrajewski/nimplex#6]
Installing nimble via codespaces for AMMap will not compile the stitching.nim. plotting.nim seemingly works correctly, but the identical method does not work for stitching.

Error: cannot open 'nimplex/src/nimplex/utils/stitching.nim'

**Comment by amkrajewski** [comment]
Hi @amr8004, thanks for testing! It looks like there have been two separate obstacles. 

The first issue, which you reported on, had to do with `AMMap` rather than `nimplex` itself - the AMMap's git submodule has been pointing to an older commit (`v0.5.1`) where `stitching.nim` utility has not been present yet (`v0.7.0`). I pushed appropriate changes and it finds the file as expected.

The second "issue" you would have gotten if you fixed the first one was that the `stitching.nim` would have expected `nimplex` installed as a `nim` library, while you only compile it as `Python` library. A quick adjustment would be to use `nimble install nimplex` in `AMMap`, but I wanted it to be a bit more straightforward so I adjusted the `nimble` branch version in a way both approaches will work.

I just tested `AMMap` Codespace and everything seems to work as expected.
